training/checkpoint.py

```diff
@@ -1,6 +1,16 @@
 import os
 import torch
-
+from typing import Optional
+from unidecode import unidecode
+
+NVIDIA_ALPHABET = ['_', '-', '!', "'", '(', ')', ',', '.', ':', ';', '?', ' ',
```

Comment:
We already have a default alphabet defined at https://github.com/BenAAndrew/Voice-Cloning-App/blob/main/training/train.py#L35. Also, will this cause problems for non-English datasets or datasets that do not contain cmudict?